snapcraft.yaml

```diff
@@ -0,0 +1,18 @@
+name: wifi-ap-client
+version: 1
+confinement: strict
+summary: HTTP over Unix
+description: |
+  This snap is implementing HTTP over Unix sockets.
+grade: stable
+
+apps:
+  unixhttpc:
```

Comment:
Lets name the app with the same name as the snap then we just have /snap/bin/wifi-ap-example-client instead of /snap/bin/wifi-ap-example-client.unixhttpc